.github/workflows/main.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
```

Comment:
We could probably use a more descriptive name.

Reply:
I left it vague as it's the only one so far and I don't know what else we'll put in there apart from unit tests but I'll rename it for you